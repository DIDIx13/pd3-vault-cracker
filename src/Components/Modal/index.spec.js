import Modal from './index.svelte';
import { render } from '@testing-library/svelte'

describe('Modal', () => {
    it('renders the modal', () => {
        const { container } = render(Modal)

        expect(container).toMatchSnapshot()
    });

    it('should be display none', () => {
        const { container } = render(Modal)

        const dialog = container.querySelector('dialog')

        expect(dialog.style.display).toBe('')
    });
});
