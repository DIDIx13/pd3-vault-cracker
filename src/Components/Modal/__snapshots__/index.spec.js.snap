// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal renders the modal 1`] = `
<body>
  <div>
    <dialog
      class="svelte-19xcnu3"
    >
      <div
        class="svelte-19xcnu3"
      >
         
        <hr />
         
         
        <button
          autofocus=""
          class="svelte-19xcnu3"
        >
          Close
        </button>
      </div>
    </dialog>
  </div>
</body>
`;
